Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/EditorTemplates/SubmissionTypeCheckboxes.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@using Resource = Resources.Areas.Administration.Problems.Views.EditorTemplates.SubmissionTypeCheckBoxes;
+@model IList<OJS.Web.Areas.Administration.ViewModels.SubmissionType.SubmissionTypeViewModel>
+
+@for (int i = 0; i < Model.Count; i++)
```

Comment:
It's not a good practice to have a loop in the views. Please use Display/EditorTemplates